supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+
+##### Geothermal
+
+Modeling of geothermal energy in GCAM is documented in [Hannam et al. (2009)](supply_energy.html#hannam2009). Like wind and solar, geothermal energy is only used in GCAM as a source of electricity production, but the quantities in the resource supply curves are indicated in terms of EJ of heat input to power plants, estimated as 10 times the quantity of electricity generated. This 10% thermo-electric efficiency is a bit lower than most regions where it has been estimated, but is the [IEA's (2011)](supply_energy.html#iea2011) default assumption. The supply curves in each region are graded, similar to the fossil resource curves but with the exception that the quantities refer to annual flows, not cumulative stocks. As with wind and solar, the supply curves are intended to capture only the portion of the costs of producing geothermal electricity that escalate with deployment, with the remainder of the costs in the corresponding technologies of the electricity sector. In the four phases of geothermal electricity production identified by [Deloitte (2008)](supply_energy.html#deloitte2008), the supply curves in GCAM include identification, exploration, and drilling, but not production. See also [geothermal](details_energy.html#geothermal)
+
+##### Hydropower
+
+Hydropower is the simplest of all energy forms in GCAM; the quantity of hydropower produced in each region and year is exogenous, prescribed as a "fixedOutput". Hydropower costs are not estimated, and the technology does not contribute to the modeled electricity price in each region. The quantities in future years are generally consistent with the long-term "economic" potential identified by the International Hydropower Association [(IHA 2000)](supply_energy.html#iha2000).
+
+##### Biomass
+
+While most of the effort in modeling biomass supply is in the [agriculture and land use component](land.html), there is a renewable resource represented in the energy system, that generally refers to municipal and industrial wastes that can be used for energy purposes. The supply curves use the same functional form as printed in the Wind section above, and the specific quantities are documented in [Gregg and Smith (2010)](supply_energy.html#gregg2010). Unlike other resources, the waste biomass supply curve is assumed to grow with GDP, as prescribed by the exogenous supply elasticity of GDP, or "gdpSupplyElast".
```

Comment:
- "use the same functional form as **printed** in the Wind section above ...".  Should this be "as **described** ..."